packages/@lightningjs/ui-components/src/components/ControlRow/ControlRow.stories.js

```diff
@@ -83,6 +83,41 @@ export const Basic = () =>
     }
   };
 
+export const Bug = () =>
+  class Bug extends lng.Component {
+    static _template() {
+      return {
+        ControlRow: {
+          type: ControlRowComponent,
+          leftControls: [],
+          contentItems: [],
+          rightControls: [],
+          lazyLoadBuffer: 1
+        }
+      };
+    }
+
+    _construct() {
+      setTimeout(() => {
+        if (this._ControlRow) {
+          this._ControlRow.addContentItems(
+            createItems(
+              3,
+              'https://image.tmdb.org/t/p/w500/frwl2zBNAl5ZbFDJGoJv0mYo0rF.jpg'
+            )
+          );
+        }
+      }, 1500);
+    }
+    _getFocused() {
+      return this.tag('ControlRow');
+    }
+
+    get _ControlRow() {
+      return this.tag('ControlRow');
+    }
+  };
+
```

Comment:
Can we remove this and include it in the PR description as instructions of how to test? Essentially just "make sure this story loads correctly"?

Reply:
Done